src/features/embalm/stepContent/components/ArchaeologistList.tsx

```diff
@@ -162,14 +168,18 @@ export function ArchaeologistList({
                     <VStack align="left">
                       <Button
                         variant="ghost"
-                        rightIcon={sortIconsMap[failsSortDirection]}
+                        rightIcon={
+                          archaeologistFilterSort.sortType === SortFilterType.FAILS
+                            ? sortIconsMap[archaeologistFilterSort.sortDirection]
+                            : sortIconsMap[SortDirection.NONE]
+                        }
```

Comment:
These 3 rightIcon routines could get refactored to a single function that takes a sort type as an arg.